lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,194 @@
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    @objc private func backButtonTapped() {
+        dismiss(animated: true, completion: nil)
+    }
+
+    private func configureNavigationController() {
+        guard let navigationController = navigationController else {
+            return
+        }
+
+        // NavigationBar Background
+
+        navigationController.navigationBar.isTranslucent = true
+        navigationController.navigationBar.barStyle = .default
+        navigationController.navigationBar.barTintColor = LucienTheme.white
+        navigationController.navigationBar.setValue(true, forKey: "hidesShadow")
+
+        // NavigationBar Title
+        navigationController.viewControllers[0].title = "Add Book"
+        navigationController.navigationBar.titleTextAttributes = [NSAttributedStringKey.font: LucienTheme.Fonts.PermanentMarker.regular.getFont(size: 30)]
+
+        // NavigationBar Back Button
+        let backButton = UIBarButtonItem(image: UIImage(named: "navBackButton"), style: .plain, target: self, action: #selector(AddComicViewController.backButtonTapped))
+        backButton.tintColor = UIColor.black
+        navigationItem.leftBarButtonItem = backButton
+
+        // NavigationBar 'Finish' Button
+        let finishButton = UIBarButtonItem(title: "Finish", style: .plain, target: self, action: nil)
+        finishButton.setTitleTextAttributes([NSAttributedStringKey.font: LucienTheme.Fonts.Muli.semiBold.getFont(size: 17)], for: .normal)
+        finishButton.tintColor = LucienTheme.finishButtonBlack
+        navigationItem.rightBarButtonItem = finishButton
+    }
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        coverPhotoLabel.font = LucienTheme.Fonts.Muli.light.getFont(size: 12)
+        coverPhotoLabel.textColor = LucienTheme.coolGrey
+
+        coverPhotoButton.titleLabel?.lineBreakMode = .byWordWrapping
+        coverPhotoButton.titleLabel?.textAlignment = .center
+        coverPhotoButton.setTitle("Take \n Cover \n Photo", for: .normal)
+        coverPhotoButton.titleLabel?.font = LucienTheme.Fonts.Muli.bold.getFont(size: 13)
+        coverPhotoButton.setTitleColor(LucienTheme.dark, for: .normal)
+        coverPhotoButton.titleEdgeInsets.top = 25
+
+        // Comic Title
+        configureLabel(label: comicTitleLabel, font: LucienTheme.Fonts.Muli.light.getFont(size: 12), color: LucienTheme.coolGrey)
```

Comment:
I would suggest splitting each section with a comment in front of it into its own function